airflow/www/views.py

```diff
@@ -282,9 +282,46 @@ def get_int_arg(value, default=0):
             paging=wwwutils.generate_pages(current_page, num_of_pages,
                                            search=arg_search_query,
                                            showPaused=not hide_paused),
-            auto_complete_data=auto_complete_data,
             num_runs=num_runs)
 
+    @expose('/dags_autocomplete')
+    @has_access
+    @provide_session
+    def dags_autocomplete(self, session):
+        arg_search_query = request.args.get('search', None)
+
+        if not arg_search_query:
+            return wwwutils.json_response({
+                'items': []
+            })
+
+        max_items_count = int(request.args.get('limit', 8))
+        filter_dag_ids = appbuilder.sm.get_accessible_dag_ids()
+
+        dags_query = session.query(DagModel.owners).filter(
+            ~DagModel.is_subdag, DagModel.is_active
```

Comment:
Did the previous view show paused dags? (i.e. is this a change in behaviour?)

Reply:
Yes. It always redirected to a view that shows all DAGs. https://github.com/apache/airflow/pull/4929/files/04953cffe59151d07a5ff44c448a57584c4247bd#diff-f38558559ea1b4c30ddf132b7f223cf9L271